.gitignore

```diff
@@ -7,6 +7,7 @@
 /phpunit.xml
 /phpcs.xml
 /vendor
+/vendors
```

Comment:
We don't use a /vendors directory.

Reply:
It is used in 2.x. Otherwise it is impossible to see changes after checkout from 2.x to 3.x in the same repo.

Reply:
How is it impossible?

Reply:
I mean to find my changes among those listed in vendors folder by git status.

Reply:
this change could be done in separated PR 
in PRs you can focus only in one concept in this case `Fix non-set precision bug in MysqlSchema::columnSq`

Reply:
Because when you are in 2.x branch you have vendors folder. When you checkout to 3.x then vendors folder is not removed as it is in gitignore in 2.x.

Reply:
your problem is very limited to only a few developers 
I and most cake developers use clean git repo in local for cake 3

Reply:
right now I have changed gitignore in my local but it is only affect my local 
I think this case is very especial case that you can do it only for your gitignore in your local IMO

Reply:
@saeideng why do you want to make contributing more difficult? I really have difficulties to understand such an attitude. I had the problem while working on `columnSql` changes. The fix was one line in gitignore. No side effects. Hurts nobody. That allowed me to fix `columnSql` faster.